includes/definitions/siteboss.yaml

```diff
@@ -0,0 +1,12 @@
+os: siteboss
```

Comment:
Is this the OS /firmware name or does it not have one and so the product name is used?

Reply:
The latter. Did not have one (that I could find) and so the product name was used.